### PR TITLE
Master

### LINKED ISSUE #4: Code Review PR#2
peer review aaron's first push to master

### LINKED ISSUE #8: Finish Ch 6 and Release Master
Update the comments in the code to your own.

Release our final version from ch 6. Pick whatever version naming scheme you want

### PR DESCRIPTION
**Pull request by awsumpter**
Submit Calculator.cpp code for peer review by Dave.

### CODE REVIEW

File: Calculator/Calculator.cpp [hunk 1]
```diff
@@ -20,24 +18,57 @@ class Token {
...
-        //not yet   case ';':    // for "print"
-        //not yet   case 'q':    // for "quit"
+    case '=': // for �print�
+    case 'x': // for �quit�
     case '(': case ')': case '+': case '-': case '*': case '/':
-        return Token(ch);        // let each character represent itself
+        return Token{ ch }; // let each character represent itself
     case '.':
     case '0': case '1': case '2': case '3': case '4':
     case '5': case '6': case '7': case '8': case '9':
```
Comment (davrdic) [inline Calculator/Calculator.cpp]: fix the brackets and indentation (lines 68-72)

File: Calculator/Calculator.cpp [hunk 2]
```diff
@@ -46,97 +77,111 @@ Token get_token()    // read a token from cin
...
+    Token t = ts.get();
     switch (t.kind) {
     case '(':    // handle '(' expression ')'
     {
         double d = expression();
-        t = get_token();
+        t = ts.get();
         if (t.kind != ')') error("')' expected");
         return d;
     }
```
Comment (davrdic) [inline Calculator/Calculator.cpp:103]: correct comment to 'n' line 99

**Review by davrdic — CHANGES_REQUESTED** [review]
couple small errors. please fix and i will approve

File: Calculator/Calculator.cpp (hunk 1 above)
Comment (awsumpter) [reply Calculator/Calculator.cpp]: These changes have been made

File: Calculator/Calculator.cpp (hunk 2 above)
Comment (awsumpter) [reply Calculator/Calculator.cpp:103]: These changes have been made

**Comment by awsumpter** [comment]
I pushed my revision commits and I can see them reflected in my main branch. Let me know if there is something else i need to do to respond to your review 👍

File: Calculator/Calculator.cpp (hunk 1 above)
Comment (davrdic) [reply Calculator/Calculator.cpp]: after the opening '{', insert a new line. see teams-bjarne dance-post for an example

**Review by davrdic — CHANGES_REQUESTED** [review]
needs one more correction to formatting

**Review by davrdic — APPROVED** [review]
looks good! go ahead and merge. we're nearing the end of chapter 7. i'm working on a draft now of the 'design' of our current build. i'll post a link to it in teams soon or put it on one drive if i finish

**Pull request merged**